### ISSUE TITLE
Package Proposal: beacons

### ISSUE THREAD

**Issue by awazgyawali** [fluttercommunity/community#21]
# Package Proposal: beacons

**Dependency name (as used in *pubspec.yaml*):** beacons
**Current pub.dev Link:** https://pub.dev/packages/beacons
**Description:** Flutter beacons plugin for iOS and Android.
**Current maintainer:** [Loup Inc., lukasz@intheloup.io AND lukaspili]
**Needs new maintainer after transfer:** [YES]
**New maintainer (if applicable):** [Aawaz Gyawali, awazgyawali@gmail.com]
**Reason for transfer:** The author has abandoned the project and there is plenty of PR that needs to be merged to make it better.

**Comment by jeroen-meijer** [comment]
Hello there,

Thanks for submitting the package proposal.

I haven't used iBeacons myself yet, and neither have I any of them to test the functionality. That's why the best I can do is a simple code review and check out other packages.

While I work on that, could you tell me the following:
1. **What does the `beacon` package offer that `flutter_beacon` (https://pub.dev/packages/flutter_beacon) doesn't?**
2. **Do you think the two can be merged into one package?** At Flutter Community, we'd love for users to have only one package to import instead of make choices regarding similar packages. This also ensures that a single package is maintained well instead of two packages where one or both might be out of date from time to time.
3. **Can you send a video showing the application in action with a beacon?**

Thank you for cooperation 😁

Best regards.

**Comment by jeroen-meijer** [comment]
Hi there,

One more thing, I found this package on pub, uploaded by the same company, `flutter_beacons` (https://pub.dev/packages/flutter_beacons). Do you know about this package, and if it can be merged, too?
All three beacons packages are extremely similar, so we'd like them to be merged before we can make a decision on wether or not to include them in the Flutter Community GitHub.

**Comment by AndruByrne** [comment]
@jeroen-meijer, I think they are looking for out-of-the-box background monitoring, which is the substantive delta between the two libraries' documentation. I see that background monitoring on flutter_beacons requires some unspecified "extra lines of code" so maybe its just a documentation problem.

**Comment by AndruByrne** [comment]
For those looking for good out-of-the-box background monitoring, do understand that BLE is a fairly hardware-specific API designed with certain uses in mind; it has a good bit of undefined space in it still (though the android beacon lib has helped a lot) https://developer.radiusnetworks.com/2013/11/13/ibeacon-monitoring-in-the-background-and-foreground.html
